lambdas/raster_tiler/lambda_function.py

```diff
@@ -37,15 +37,42 @@ def array_to_img(arr: np.ndarray) -> str:
 
 def get_tile_array(src_tile: str, window: Window) -> np.ndarray:
     """Create mercator tile from GFW WM Tile Set images."""
-
-    with rasterio.open(src_tile) as src:
-        profile = src.profile
-        bands = profile["count"]
-        indexes = tuple(range(1, bands + 1))
-        out_shape = (len(indexes), TILE_SIZE, TILE_SIZE)
-        data = src.read(
-            window=window, boundless=True, out_shape=out_shape, indexes=indexes
-        )
+    # if running lambda in localstack, need to use special docker IP address provided in env to reach localstack
+    LOCALSTACK_HOSTNAME = os.environ.get("LOCALSTACK_HOSTNAME", None)
+    AWS_ENDPOINT_HOST = f"{LOCALSTACK_HOSTNAME}:4566" if LOCALSTACK_HOSTNAME else None
```

Comment:
Best to create a seperate function which sets the globals based on the environment